lib/lotus/validations/error.rb

```diff
@@ -63,21 +60,16 @@ class Error
 
       # Initialize a validation error
       #
-      # @param attribute_name [Symbol] the name of the attribute
       # @param validation [Symbol] the name of the validation
       # @param expected [Object] the expected value
       # @param actual [Object] the actual value
-      # @param namespace [String] the optional namespace
       #
       # @since 0.1.0
       # @api private
-      def initialize(attribute_name, validation, expected, actual, namespace = nil)
-        @attribute_name = attribute_name.to_s
+      def initialize(validation, expected, actual)
```

Comment:
Actually you don't need to have flattened collection of errors, you can use:
```ruby
validator.errors.each { |attr, errors| }
```
And in `attr` you have a reference with the attribute. That's why I removed it from the error object - because of redundant information. Probably I'm missing some downsides of my approach. What do you think about this? I'd love to hear some valid points against it before restoring it.